Cargo.toml

```diff
@@ -18,19 +22,19 @@ serde_indextree = "0.2.0"
 serde_json = "1.0.88"
 thiserror = "1.0.37"
 typetag = "0.2.3"
+wgpu = { version = "0.14.2", optional = true }
+tracing = "0.1.37"
+tracing-subscriber = "0.3.16"
 
 [dev-dependencies]
 clap = { version = "4.0.26", features = ["derive"] }
-glutin = "0.30.3"
-glutin-winit = "0.2.1"
-raw-window-handle = "0.5.0"
-tracing = "0.1.37"
-tracing-subscriber = "0.3.16"
-winit = "0.27.5"
+
 
 [features]
-default = ["opengl"]
+default = ["gl-winit", "wgpu"]
 opengl = ["dep:glow"]
+gl-winit = ["opengl", "glutin", "glutin-winit", "winit"]
+wgpu = ["dep:wgpu"]
```

Comment:
Hmmmm... I wonder if there's a better way to organize these features. This feels a bit messy.

Also I don't think `gl-winit` and `wgpu` should be both toggled at the same time. I think ideally they should be exclusive, if that makes sense.

Reply:
I enabled all renderers for easier development. Please remind me to remove them before merging into main.

Additionally, these lines of changes were included in this PR by mistake. It should have been in #6.

Reply:
_woops :v_

Reply:
All the renderers are enabled for development purposes. Please remind me to fix this before merging.

Also, it should have gone to #6 instead. Sorry!